package.json

```diff
@@ -8,7 +8,7 @@
     "compile": "tsc --build tsconfig.build.json",
     "compile:clean": "npm run compile -- --clean",
     "watch": "npm run compile && tsc --build tsconfig.json --watch",
-    "version": "npm i --package-lock-only && node ./prompt-update-changelogs.js",
+    "version": "npm i --package-lock-only && git add package-lock.json",
```

Comment:
I'm a little stumped as well. Reading through the docs, I would've been surprised if it adds the root `package-lock.json` because lerna is not expecting any changes in `package.json` since we use independent versioning. There are several lerna issues that seems like adding `package-lock.json` for the root could've been considered a bug, so I wonder if this was unintended behavior that got fixed along the way.